anesthetic/labelled_pandas.py

```diff
@@ -46,22 +48,35 @@ def cmp(x, y):
 
     results.sort(key=cmp_to_key(cmp))
 
-    for s in results:
+    for s, l in results:
         if s is not None:
+            if hasattr(s, "name"):
+                try:
+                    s.name = l[s.name]
+                except Exception:
+                    pass
```

Comment:
This is the business end of the change in the function, which now expects a list of pairs of (function, labelmap) to be passed, and then tries to replace the name if it can.

Reply:
pandas esotericism means that any of `ValueError`, `TypeError`, `KeyError` or `IndexingError` are possible (and covered in various subsets). Still, this is better than `Exception`, so I've updated this in 5267cb7